src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -161,11 +161,14 @@ public bool PreAuthenticate
             get { return true; }
             set
             {
+                /*
+                TODO:#18104
```

Comment:
@davidsh should comment.  But it looks like both the WinHttp and Curl implementations default to false, yet the getter above is hardcoded to true.  Couldn't this be addressed just be changing the getter above to return false instead, thereby matching what's done in the other implementations (and then if something is setting Preauthenticate to false and that's what's throwing for you, it'll no longer throw, because `value != PreAuthenticate` will be false. Anyway, I realize it's temporary.  I just want to make sure we're moving in the right direction.